lib/index.ts

```diff
@@ -76,7 +76,7 @@ async function buildDepTree(
 
   const manifestFile: ManifestFile = parseManifestFile(manifestFileContents);
   if (!manifestFile.name) {
-    manifestFile.name = defaultManifestFileName;
+    manifestFile.name = path.basename(defaultManifestFileName);
```

Comment:
how is changing the name going to affect existing projects?

Reply:
Good catch! Fixed